pkgs/development/tools/parsing/flex/default.nix

```
@@ -13,11 +17,11 @@ stdenv.mkDerivation rec {
 
   # Also upstream, will be part of 2.6.5
   # https://github.com/westes/flex/commit/24fd0551333e
-  patches = [(fetchpatch {
+  patches = [(fetchurl {
     name = "glibc-2.26.patch";
     url = "https://raw.githubusercontent.com/lede-project/source/0fb14a2b1ab2f82c"
         + "/tools/flex/patches/200-build-AC_USE_SYSTEM_EXTENSIONS-in-configure.ac.patch";
```

This should use the full hash.

```suggestion
    url = "https://raw.githubusercontent.com/lede-project/source/0fb14a2b1ab2f82ce63f4437b062229d73d90516/tools/flex/patches/200-build-AC_USE_SYSTEM_EXTENSIONS-in-configure.ac.patch";
```